### ISSUE TITLE
Offline Whisper mode

### ISSUE THREAD

**Issue by nick-harder** [lspahija/VoxGPT#9]
Would it be possible to use a local version of the whisper instead of using the API? As far as I know it is available in open source.

**Comment by adocampo** [comment]
Yes, and it can handle multiple languages, no need to use another STT
